src/app_service/service/network/service.nim

```diff
@@ -136,3 +139,18 @@ proc getNetworkForCollectibles*(self: Service): NetworkDto =
     return self.getNetwork(Goerli)
 
   return self.getNetwork(Mainnet)
+
+proc updateNetworkEndPointValues*(self: Service, chainId: int, newMainRpcInput, newFailoverRpcUrl: string) =
+  let network = self.getNetwork(chainId)
+
+  if network.rpcURL == newMainRpcInput and network.fallbackURL == newFailoverRpcUrl:
```

Comment:
@Khushboo-dev-cpp do we want to return as well in case `newMainRpcInput == newFailoverRpcUrl`?

Reply:
hmmm interesting use case, I am not so sure, but if a user wants to set the same as fail over and default i guess we can let the user do that?

Reply:
What's the point in setting the same fallback server? If server A fails, then server B will fail for sure if A and B are the same. What has some sense is to either set fallback server to a different address or leave it empty. (though that's my opinion)

Reply:
Multiple field validation are tricky and requires careful edge case situation to be dealt with.
e.g I want to reverse fallback and main but I save in between? why should not I be able to do this?

I think this is not worth it to prevent this. After all this is user decision and doesn't break the app. I wouldn't block on this but only offers a warning if both are identical.

We can add a follow up issue with very low priority

Reply:
Agree with you @alaibe created https://github.com/status-im/status-desktop/issues/11551 for follow up. Need design to add a warning message first